### ISSUE TITLE
Wrong version number 2.1.8 is 2.1.7 (comment in js file)

### ISSUE THREAD

**Issue by imkebe** [Semantic-Org/Semantic-UI#3621]
~~~
 /*
 * # Semantic UI - 2.1.7
 * https://github.com/Semantic-Org/Semantic-UI
 * http://www.semantic-ui.com/
 *
 * Copyright 2014 Contributors
 * Released under the MIT license
 * http://opensource.org/licenses/MIT
 *
 */
~~~

**Comment by jlukic** [comment]
Seems to only affect `semantic.js` not sure cause

**Comment by jlukic** [comment]
This will be fixed in `2.1.9`.
